### DISCUSSION TITLE
Compatability with vim-illuminate

### DISCUSSION THREAD

**Discussion by nianiam** [Bekaboo/dropbar.nvim#162] [Q&A] [answered]
Does anyone know how I might go about disabling the Dropbar buffers from Vim-illuminate?

It matches the `>` character under the cursor in the menus:

<img width="269" alt="image" src="https://github.com/Bekaboo/dropbar.nvim/assets/70338628/7433767c-1d8e-40eb-8618-b8069adce9f0">

From their [docs](https://github.com/RRethy/vim-illuminate?tab=readme-ov-file#configuration) you can disable it in a number of ways: `filetype`, `mode` and `providers`. However, I am unsure how to figure out what Dropbar's values for these are.

**Comment by Bekaboo** [comment]
Seems that you want to disable vim-illuminate in dropbar buffers? If so, it's better to ask vim-illuminate's maintainer for help, since I don't know how vim-illuminate works.

**Reply by nianiam** [reply]
Fair enough. Although the repo has no discussions and this isn’t exactly an “issue” 😅

Does Dropbar name or identify the buffers in any way? There might be a way of using the: `should_enable = function(bufnr) … end` option to get buffer information from the buffer number and then disable it for Dropbar’s buffers.

**Reply by Bekaboo** [reply]
Dropbar buffer has filetype 'dropbar_menu' (for normal menu buffers) or 'dropbar_menu_fzf' (for fzf buffers).

**Reply by nianiam** [reply, marked as answer]
Thanks. With that it was easy to disable it with the following:

```lua
-- Using lazy.nvim
return {
  "RRethy/vim-illuminate",
  opts = {
    filetypes_denylist = {
      ...
      "dropbar_menu",
      "dropbar_menu_fzf",
    },
  },
}
```
